src/algebraic_geometry/is_open_comap_C.lean

```diff
@@ -0,0 +1,108 @@
+lemma is_prime_C_map_of_is_prime (H : is_prime P) : is_prime (map C P : ideal (polynomial R)) :=
+(quotient.is_integral_domain_iff_prime (map C P : ideal (polynomial R))).mp
+  (is_integral_domain_C_map_quotient H)
+
+/-- Given a polynomial `f ∈ R[x]`, `image_of_Df` is the subset of `Spec R` where at least one
+of the coefficients of `f` does not vanish.  We will prove that `image_of_Df` is the image of
+`(zero_locus {f})ᶜ` under the morphism `comap C : Spec R[x] → Spec R`. -/
+def image_of_Df (f : polynomial R) := {p : prime_spectrum R | ∃ i : ℕ , (coeff f i) ∉ p.1}
+
+lemma std_aff_is_open (a : R) : is_open ({p : prime_spectrum R | a ∉ p.1 }) :=
+begin
+  refine ⟨{a} , ext (λ x, ⟨λ xh, _,λ xh,  _⟩)⟩;
+  simpa only [mem_zero_locus, not_not, mem_set_of_eq, singleton_subset_iff, mem_compl_eq] using xh,
+end
+
+lemma is_open_image_of_Df (f : polynomial R) : is_open (image_of_Df f) :=
+begin
+  rw [image_of_Df, set_of_exists (λ i (x : prime_spectrum R), coeff f i ∉ x.val)],
+  exact is_open_Union (λ i, std_aff_is_open (coeff f i)),
+end
+
+lemma mem_span_C_coeff (f : polynomial R) :
+  f ∈ span {tt : polynomial R | ∃ i : ℕ ,tt = (C (coeff f i))} :=
+begin
+  conv_lhs {rw as_sum_support f},
+  refine submodule.sum_mem _ (λ i hi, _),
+  rw [← C_mul_X_pow_eq_monomial, mul_comm],
+  exact submodule.smul_mem _ _ (submodule.subset_span ⟨i, rfl⟩),
+end
+
+lemma span_subset_of_coeff_mem_comap {f : polynomial R} {I : ideal (polynomial R)}
+  (hfi : ∀ (i : ℕ), f.coeff i ∈ (comap (C : R →+* polynomial R) I)) :
+  (span {tt : polynomial R | ∃ (i : ℕ), tt = C (f.coeff i)}).carrier ⊆ I :=
+begin
+  refine bInter_subset_of_mem _,
+  rintros _ ⟨i, rfl⟩,
+  exact (submodule.mem_coe I).mpr (hfi i),
+end
+
+/-- The image of `Df` is contained in an open set (with which it actually coincides). -/
+lemma exists_coeff_not_mem_C_comap {f : polynomial R} {I : prime_spectrum (polynomial R)} :
+  f ∉ I.1 → ∃ i : ℕ , coeff f i ∉ (comap (C : R →+* polynomial R) I).1 :=
+begin
+  contrapose!,
+  exact λ hfi, (span_subset_of_coeff_mem_comap hfi) (mem_span_C_coeff f),
+end
+
+/-- If a point of `Spec R[x]` is not contained in the vanishing set of `f`,
+then its image in `Spec R` is contained in the open where at least one of the coefficients
+of `f` is non-zero. This lemma is a reformulation of `exists_coeff_not_mem_comap_C`. -/
+lemma C_comap_mem_image_of_Df {f : polynomial R} {I : prime_spectrum (polynomial R)}
+  (H : I ∈ (prime_spectrum.zero_locus {f} : set (prime_spectrum (polynomial R)))ᶜ ) :
+  prime_spectrum.comap (polynomial.C : R →+* polynomial R) I ∈ image_of_Df f :=
+begin
+  refine exists_coeff_not_mem_C_comap (λ h, (H ((I.mem_zero_locus {f}).mp _))),
+  rintro _ ⟨rfl⟩,
+  exact h,
+end
+
+-- to src/data/set/lattice, PR#5691.
+-- If this PR is merged into master, then this lemma can be removed
+lemma bUnion_of_singleton_of_coe {α : Type*} (s : set α) :
+  (⋃ (i : s), {i} : set α) = s :=
+ext $ by simp
```

Comment:
The usual practice here actually is not to mention other PR numbers in the code like this, but instead to put the commits of one PR on top of the commits of the other(s), as (I think) you did before.

In any case, since we cannot merge this PR until #5691 is resolved, this PR actually does have a dependency on that one, even though this branch may not have the commits of that one.

Reply:
Bryan, thank you again for fixing all my mistakes!  I did not set up the dependency, since I am not entirely comfortable with Git/leanproject and bors intimidates me a little...

Anyway, I am glad that now the dependency is there!